firebase_admin/db.py

```diff
@@ -678,8 +687,11 @@ def from_app(cls, app):
 
     def request(self, method, urlpath, **kwargs):
         resp_headers = kwargs.pop('resp_headers', False)
+        params = kwargs.get('params', None)
         resp = self._do_request(method, urlpath, **kwargs)
-        if resp_headers:
+        if resp_headers and params == 'print=silent':
```

Comment:
Can we just return `resp.json(), resp.headers` in this case? Does that trigger an error? If not I'd say simply return the tuple, and ignore the payload value (which I assume would get set to None).